### PR TITLE
Update the return type of `convert_qubo_to_model`

### PR DESCRIPTION
**Pull request by vtomole**
Fixes https://github.com/SupertechLabs/general-superstaq/issues/29

### CODE REVIEW

**Comment by vtomole** [comment]
@hallben4 Please review.

**Review by hallben4 — APPROVED** [review]
lgtm

**Comment by vtomole** [comment]
Thanks!!

File: general_superstaq/qubo.py [hunk 1]
```diff
@@ -16,7 +16,7 @@ def read_json_qubo_result(json_dict: dict) -> np.recarray:
     return gss.converters.deserialize(json_dict["solution"])
 
 
-def convert_qubo_to_model(qubo: qv.QUBO) -> List[Dict[str, Any]]:
+def convert_qubo_to_model(qubo: qv.QUBO) -> List[Dict[str, Iterable[str]]]:
```
Comment (richrines1) [inline general_superstaq/qubo.py:19]: is this annotation correct? i think the "value" entry is just a number, not an iterable

Comment (richrines1) [reply general_superstaq/qubo.py:19]: also out of curiosity why isn't our model just `dict(qubo)`

Comment (vtomole) [reply general_superstaq/qubo.py:19]: It's not correct.  That's why we didn't do a deployment last week. I meant to follow up on this but didn't get to.

Comment (richrines1) [reply general_superstaq/qubo.py:19]: ah cool sorry still out of the loop. also realizing `dict(qubo)` doesn't work because it isn't json serializable

**Pull request merged**